topics/go/algorithms/numbers/palindrome/palindrome.go

```diff
@@ -5,17 +5,17 @@ import "github.com/ardanlabs/gotraining/topics/go/algorithms/numbers/reverse"
 // Is checks if a integer is a Palindrome.
 func Is(input int) bool {
 
-	// Integer below zero cannot be palindrome.
+	// An negative integer is not a palindrome.
```

Comment:
A negative integer.

An when the next word starts with a vowel.

A when it doesn't.

Reply:
Fixed